skipper/utils.py

```diff
@@ -137,3 +137,20 @@ def image_to_dockerfile(image):
 
 def dockerfile_to_image(dockerfile):
     return dockerfile.replace('Dockerfile.', '')
+
+
+def get_runtime_command():
+    global CONTAINER_RUNTIME_COMMAND
```

Comment:
better take it from ENV or config? Why hardcoded in the code?
Also, you should assume the server have both docker and podman installed

Reply:
are u sure u cand have 2 of them?